test/system-node.js

```diff
@@ -0,0 +1,50 @@
+import { System as globalSystem, setBaseUrl, applyImportMap } from '../src/system-node.js';
+import { resetBaseUrl } from '../src/common.js';
```

Comment:
How are we testing the `dist` build here? Ideally that should be inlcuded somehow.

Reply:
👍 am happy to switch to changing the dist build - I started with that but changed it to src to match the system-core test